object/id/address.go

```diff
@@ -181,3 +182,9 @@ func (x Address) String() string {
 func (x Address) IsZero() bool {
 	return x.cnr.IsZero() && x.obj.IsZero()
 }
+
+// Compare performs three-way comparison of two addresses and can be used
+// for address list sorting.
+func (x Address) Compare(other Address) int {
```

Comment:
not sure i understand this func

Reply:
i meant sorting adresses. do you have a use case?

Reply:
https://github.com/nspcc-dev/neofs-node/blob/master/pkg%2Flocal_object_storage%2Fengine%2Fselect_test.go#L16